lms/djangoapps/ccx/views.py

```diff
@@ -93,6 +93,12 @@ def wrapper(request, course_id):
 
         if not course.enable_ccx:
             raise Http404
+        elif configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and not run_extension_point(
```

Comment:
Please, add a comprehensive comment to understand why this is here. @anfbermudezme